extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js

```diff
@@ -97,7 +97,7 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
     }
 
     $scope.getMinText = function(item) {
-        if (!item.stateDescription || !item.stateDescription.minimum) {
+        if (!item.stateDescription || isNaN(item.stateDescription.minimum)) {
```

Comment:
This function should return a text, it does not need to check if the text is a valid number or something similar.

If <code>item.stateDescription.minimum</code> is defined but an empty string, the first test does not match.
So, the functions continues.
If a pattern is defined, the pattern should be used to parse the (perhaps empty) minimum string.
If no pattern is defined, the minimum string should be returned (also okay if it is empty -- an empty string is returned in the first branch).
At least the function does return a non-null min text.